<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>281</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>46</height>
      </rect>
     </property>
     <property name="title">
      <string>Информация о герое</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="heroName">
        <property name="text">
         <string>Гор</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Уровень здаровья:</string>
     </property>
     <widget class="QProgressBar" name="healthLevel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>276</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Список предметов</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="subjectsList"/>
      </item>
      <item>
       <widget class="QPushButton" name="useItem">
        <property name="text">
         <string>Использовать предмет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openShop">
        <property name="text">
         <string>Купить что-нибудь</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Деньги:</string>
     </property>
     <widget class="QSpinBox" name="money">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Описание комнаты</string>
    </property>
    <widget class="QTextBrowser" name="roomDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="roomName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Комната:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="west">
       <property name="text">
        <string>Запад</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="east">
       <property name="text">
        <string>Восток</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="south">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Юг</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="north">
       <property name="text">
        <string>Север</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Прдметы комнаты</string>
    </property>
    <widget class="QListWidget" name="roomItems">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="peekItem">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Подобрать предмет</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
